api/src/authkeys/create.ts

```diff
@@ -29,6 +30,7 @@ export async function createAuthKey(
   const jwt = await new SignJWT({
     '_couchdb.roles': user.roles ?? [],
     name: user.name,
+    server: CONDUCTOR_PUBLIC_URL,
```

Comment:
do we use issuer yet - that's a more standard way to describe the server that issued a token - see https://auth0.com/docs/secure/tokens/json-web-tokens/json-web-token-claims and https://www.iana.org/assignments/jwt/jwt.xhtml#claims